Description: Set class name
 The default class name for Tk application is "Tk".
 .
 This patch sets it to "Rhinote", so the correct application name
 is displayed in such places as the GNOME Shell top bar.
Author: Andrea Bolognani <eof@kiyuko.org>
Forwarded: Marv Boyes <greyspace@tuxfamily.org>
Last-Update: 2012-02-01
Index: rhinote.git/rhinote.py
===================================================================
--- rhinote.git.orig/rhinote.py
+++ rhinote.git/rhinote.py
@@ -29,7 +29,7 @@ import subprocess
 
 # the root window:
 def Rhinote():
-    r = Tkinter.Tk()
+    r = Tkinter.Tk(className = 'Rhinote')
     r.option_add('*font', '{Helvetica} 11')
     t = TextWidget(r, bg = '#f9f3a9', wrap = 'word', undo = True)
     t.focus_set()
